sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3705,6 +3705,24 @@ public partial class EncryptionSettingsGroup
         public static bool operator !=(Azure.ResourceManager.Compute.Models.ExpandTypesForListVm left, Azure.ResourceManager.Compute.Models.ExpandTypesForListVm right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct FileFormat : System.IEquatable<Azure.ResourceManager.Compute.Models.FileFormat>
```

Comment:
This name is quite generic. Could we be more specific?

Reply:
Should I change name in rename-rules section or Are you requesting a name change in Swagger?

Reply:
Sorry I did not see this message - but renaming things should be done in `rename-mapping` (not rename-rules)